package.json

```diff
@@ -27,6 +27,8 @@
     "react": "^19.1.0",
     "react-dom": "^19.1.0",
     "react-router": "^7.8.0",
+    "rrweb": "^2.0.0-alpha.4",
+    "rrweb-player": "^1.0.0-alpha.4",
```

Comment:
Looking at this [link](https://github.com/rrweb-io/rrweb/blob/master/packages/rrweb-player/package.json#L3), the rrweb-player version can also be the same as rrweb itself. Why does it have a different version?